src/app/content/hooks/locationChange/resolveContent.ts

```diff
@@ -155,7 +155,7 @@ const getBookInformation = async(
   services: AppServices & MiddlewareAPI,
   pageId: string
 ) => {
-  const devEnvironment = APP_ENV === 'development';
+  const devEnvironment = UNLIMITED_CONTENT;
```

Comment:
please just replace your usage on line 170 rather than aliasing this variable like this